lib/facade/MonitoringFacade.ts

```diff
@@ -193,18 +200,6 @@ export class MonitoringFacade extends MonitoringScope {
   // GENERIC
   // =======
 
-  createdDashboard(): Dashboard | undefined {
```

Comment:
We'll want to keep these methods, otherwise it's a breaking change.

We can just make them call `this.dashboardFactory?.getDashboard(DefaultDashboards.RELEVANT_NAME)` and mark them as `@deprecated` and point towards calling that directly though, then remove them later.